rmf_traffic/src/rmf_traffic/Conflict.cpp

```diff
@@ -121,54 +121,116 @@ std::vector<ConflictData> DetectConflict::between(
 }
 
 //==============================================================================
-bool DetectConflict::broad_phase(
-    const Trajectory& trajectory_a,
-    const Trajectory& trajectory_b)
+
+namespace {
+
+using BoundingBox = std::pair<Eigen::Vector2d, Eigen::Vector2d>;
```

Comment:
I recommend defining structs with explicit fields, because it's hard to remember what each index means or what first/second refers to. For example:

```
struct BoundingBox
{
  Eigen::Vector2d min;
  Eigen::Vector2d max;
};
```

Reply:
This greatly improves readability of the code! Thanks for the suggestion.